build-tools/build.gradle

```diff
@@ -37,6 +37,10 @@ gradlePlugin {
             id = 'elasticsearch.esplugin'
             implementationClass = 'org.elasticsearch.gradle.plugin.PluginBuildPlugin'
         }
+        stableEsPlugin {
+          id = 'elasticsearch.stable_esplugin'
```

Comment:
our given convention is to use - instead of _ so it should be `elasticsearch.stable-esplugin`